### DISCUSSION TITLE
What does adnanh webhook give you that Express doesn't?

### DISCUSSION THREAD

**Discussion by ghost** [adnanh/webhook#659] [Q&A]
I have used this webhook project before and it worked great. But researching a new project build and keptgetting recommended Express to process webhooks.

I will mostly be executing Node files and working with SSH on a AWS Lightsail server with the payloads.

What does adnanh webhook offer that the most popular webhook platforms such as Express don't?

**Comment by ghost** [comment]
"You can use Express.js to handle HTTP requests and **trigger SSH shell commands** on an AWS Lightsail server to execute FFmpeg commands or any other shell commands. To achieve this, you can utilize the child_process module in Node.js, which allows you to execute shell commands from your Node.js application."

**Reply by moorereason** [reply]
I don't like complicated runtimes (Node.js, python, etc), so I prefer webhook's single binary.
